lib/aws_auditor/commands/audit.rb

```diff
@@ -9,6 +9,6 @@
   c.action do |global_options, options, args|
     require_relative '../scripts/audit'
     raise ArgumentError, 'You must specify an AWS account' unless args.first
-    AwsAuditor::Scripts::Audit.execute(args.first,options)
+    AwsAuditor::Scripts::Audit.execute(args, options)
```

Comment:
You should add the new argument as a flag, which I believe gets populated in the options as a hash.